source/agg-pipeline-builder/export-pipeline-results.txt

```diff
@@ -0,0 +1,64 @@
+.. _export-agg-builder-results:
+
+===================================
+Export Aggregation Pipeline Results
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+To use your aggregation data in other external tools, you can export your 
+pipeline results from |compass| as a JSON or CSV file. You can export your 
+pipeline results at any point while creating or editing your pipeline.
```

Comment:
The main point I was trying to drive was that you don't need to have a completed pipeline to be able to export results, but on second look I see how that sentence as is looks a bit confusing. Opting to remove it since it doesn't seem crucial to the user experience to know this.